website/content/en/docs/concepts/node-templates.md

```diff
@@ -583,6 +583,20 @@ Write-Host "Running custom user data script"
 </powershell>
 ```
 
+{{% alert title="Note" color="primary" %}}
```

Comment:
To be consistent with other Windows docs, please set the title to 'Windows Support Notice' and the color to warning

Reply:
thanks, updated.